src/components/WorkspacesGettingStarted/Card.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import '@testing-library/jest-dom';
```

Comment:
We moved `import '@testing-library/jest-dom';` to a [test setup file](https://github.com/openshift/hac-infra/blob/main/setup-react.ts) so it doesn't need to be included on tests anymore.

Doesn't hurt if it is there - just isn't needed anymore :-)